.circleci/config.yml

```diff
@@ -208,6 +213,23 @@ jobs:
           command: ./gradlew --no-daemon containerTests -i
       - capture_test_results
 
+  acceptanceTestsGoQuorum:
+    parallelism: 1
+    executor: acceptance_tests_executor
+    steps:
+      - prepare
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: GoQuorum Acceptance Tests (failures ignored)
+          no_output_timeout: 40m
+          command: ./gradlew --no-daemon acceptanceTestsGoQuorum || true # ignore result of job for now, until all issues are fixed
```

Comment:
@macfarla @pinges is the plan to merge this PR and then you work on fixing the issues or is to merge this PR after all issues are fixed?